ckanext/publicamundi/plugins.py

```diff
@@ -1072,6 +1073,10 @@ def get_auth_functions(self):
         funcs = {
             'dataset_translation_update': 
                 ext_actions.package.dataset_translation_check_authorized,
+            'member_create':
```

Comment:
These authorization hooks should be moved to `Dataset` plugin, since they are not related to multilinguality.